src/containerDeployment/dockerUtils.ts

```diff
@@ -473,6 +494,13 @@ export async function checkIfContainerIsReadyForConnections(
                 if (readyLine) {
                     clearInterval(interval);
                     dockerLogger.appendLine(`${containerName} is ready for connections!`);
+                    sendActionEvent(
+                        TelemetryViews.ContainerDeployment,
+                        TelemetryActions.StartContainer,
+                        {
+                            timeToStartInMs: `${Date.now() - start}`,

```

Comment:
stuff that you might want to math over (like this) should part of the "measures" bag (the next parameter, iirc) rather than the "properties" bag